app/store/db.js

```diff
@@ -15,6 +18,39 @@ db.version(1).stores({
   nodes: 'id'
 })
 
+// Migrate custom wallets to lndconnect.
+db.version(2).upgrade(tx =>
+  tx.wallets.toCollection().modify(wallet => {
+    if (wallet.type === 'local') {
```

Comment:
Could be a good idea to switch to `switch` or use `else` more consistently (there is one `if` and one `else if`